### ISSUE TITLE
Upgrade to XDebug 3

### ISSUE THREAD

**Issue by roborourke** [humanmade/altis-local-server#242]
XDebug 3 is out with many improvements including negligible performance impact when not using its features so it could be an "always on" feature potentially.

https://xdebug.org/announcements/2020-11-25

**Comment by nathanielks** [comment]
Xdebug 3.0.1 is included in our PHP 7.4 image 😃 :
```
/usr/src/app # pecl list
Installed packages, channel pecl.php.net:
=========================================
Package   Version State
apcu      5.1.19  stable
igbinary  3.1.6   stable
imagick   3.4.4   stable
mcrypt    1.0.3   stable
memcached 3.1.5   stable
msgpack   2.1.2   stable
redis     5.3.2   stable
ssh2      1.2     beta
xdebug    3.0.1   stable
zip       1.19.2  stable
```

**Comment by nathanielks** [comment]
I've also tagged [3.4.0](https://github.com/humanmade/docker-wordpress-php/releases/tag/3.4.0) with our latest changes to PHP 7.2. I'm waiting for that build to complete to verify if it also includes the latest Xdebug.

**Comment by nathanielks** [comment]
3.4.0 also includes the latest Xdebug 😄 
```
Installed packages, channel pecl.php.net:
=========================================
Package  Version State
apcu     5.1.19  stable
igbinary 3.1.6   stable
imagick  3.4.4   stable
mcrypt   1.0.1   snapshot
msgpack  2.1.2   stable
redis    5.3.2   stable
ssh2     1.1.2   alpha
v8js     2.1.0   stable
xdebug   3.0.1   stable
zip      1.19.2  stable
```

**Comment by roborourke** [comment]
Nice! Thanks man

**Comment by roborourke** [comment]
Closed by #244
